contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV1.sol

```diff
@@ -372,15 +387,19 @@ contract MinterMerkleV1 is ReentrancyGuard, IFilteredMinterMerkleV0 {
         );
 
         // limit mints per address by project
-        if (projectMintedBy[_projectId][msg.sender]) {
-            require(
-                _projectConfig.mintLimiterDisabled,
-                "Limit 1 mint per address"
-            );
-        } else {
-            // EFFECTS
-            projectMintedBy[_projectId][msg.sender] = true;
-        }
+        uint256 _maxProjectInvocationsPerAddress = _projectConfig
+            .useMaxInvocationsPerAddressOverride
+            ? _projectConfig.maxInvocationsPerAddressOverride
+            : DEFAULT_MAX_INVOCATIONS_PER_ADDRESS;
+
+        // FINAL CHECK, WITH FOLLOW-ON EFFECTS
+        require(
+            // _++ returns current invocations, then increments
+            projectUserInvocations[_projectId][msg.sender]++ <
```

Comment:
Nit: Does doing incrementing and reading in this way having meaningful gas savings associated with it? I find this more confusing to read than separating out the ops re: order of operations here and wonder if it is worth keeping more readable.

Reply:
(same response as above - broke out to separate statement for readability)